iothub/device/src/Transport/DeviceIdentity.cs

```diff
@@ -62,6 +62,11 @@ private static string CreateAudience(IotHubConnectionString connectionString)
             }
         }
 
+        internal bool IsPooling()
+        {
+            return AuthenticationModel != AuthenticationModel.X509 && (AmqpTransportSettings?.AmqpConnectionPoolSettings?.Pooling ?? false);
```

Comment:
Nit: coding standards - please add () around the first operand as well.

Reply:
done